strawberry/aiohttp/views.py

```diff
@@ -86,16 +90,19 @@ def __init__(self, request: web.Request, ws: web.WebSocketResponse) -> None:
         self.request = request
         self.ws = ws
 
-    async def iter_json(self) -> AsyncGenerator[Dict[str, object], None]:
+    async def iter_json(
+        self, ignore_parsing_errors: bool
```

Comment:
nit: ensure this needs to be passed as a keyword to avoid someone calling as `iter_json(True)` and the `True` there is hard to understand the meaning

```suggestion
        self,
        *,
        ignore_parsing_errors: bool,
```

Reply:
Also, should it have a default value? (not sure, just asking =P)

Reply:
Whoops. For some reason, I always thought keyword-only args were too new for the oldest Python version Strawberry supports. But they're not. I just made them keyword-only and also added a default.
I first didn't add a default to make it more "explicit," but `False` is a sane default here.